pkg/cmd/server/api/types.go

```diff
@@ -284,6 +284,9 @@ type GoogleIdentityProvider struct {
 	ClientID string
 	// ClientSecret is the oauth client secret
 	ClientSecret string
+
+	// HostedDomain is the optional Google App domain to restrict logins to
+	HostedDomain string
```

Comment:
I couldn't find a description of the format, so I didn't want to assume standard subdomain format